src/main/java/school/lemon/changerequest/java/io/pr1/task2/NumberSwapEngineImp.java

```diff
@@ -0,0 +1,107 @@
+package school.lemon.changerequest.java.io.pr1.task2;
+
+import java.io.*;
+
+/**
+ * Created by User on 03.02.2017.
+ */
+public class NumberSwapEngineImp implements NumberSwapEngine {
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public boolean swapMinAndMax(String filename) throws FileNotFoundException {
+        File file = new File(filename);
+        if (!file.exists()) {
+            throw new FileNotFoundException();
+        }
+        BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+        int min = 0;
+        int countMin = 0;
+        int max = 0;
+        int countMax = 0;
+        int temp;
+
+        boolean firstMin = true;
+        String[] result = null;
+        String[] strins = null;
+        try {
+            String line = bufferedReader.readLine();
+            if (line == null) {
+                return true;
+            } else if (line.length() < 2) {
+                return true;
+            }
+            strins = line.split(SEPARATOR);
+            result = strins.clone();
+            for (int i = 0; i < strins.length; i++) {
+                temp = Integer.parseInt(strins[i].trim());
+                while (firstMin) {
```

Comment:
Why do you need loop here? It's enough to set min as first parsed value in array.
It's pretty good, that you've done all these in one loop, but it looks very ugly. So I'd prefer to split this into several loops, e.g. first one will parse all integer values, second one - will find min/max values etc.
Also please try to move these huge logic to some other method, thus code will be much more readable.